### DISCUSSION TITLE
Github Action Rejected

### DISCUSSION THREAD

**Discussion by Potential17** [community/community#26154] [Actions]
<p>Hey,</p>
<p>My submission got rejected by saying that it is not a usable action,</p>
<p>Can anyone specify me the exact problem in my repo and what changes needs to be done in that.</p>
<p>Repo Link</p>
<p><a href="https://github.com/Potential17/Twitter-retweet-bot" rel="nofollow noopener">https://github.com/Potential17/Twitter-retweet-bot</a></p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Did you try to submit your action to <a href="https://githubhackathon.com/" rel="nofollow noopener">https://githubhackathon.com/</a>? If so, please ask your question here: </p>
<p><a href="https://github.community/t/featured-event-github-actions-hackathon/13091">https://github.community/t5/Events/Featured-Event-GitHub-Actions-Hackathon/td-p/48206</a> Thank you for your coorperation. </p>
